### ISSUE TITLE
Delly shouln't use INFO/END for the BND / TRANSLOC

### ISSUE THREAD

**Issue by lindenb** [dellytools/delly#159]
Hi,
unless I'm wrong Delly2 uses the `INFO/END` attribute to specifiy the second position of the breakpoint.  This breaks the integrity of the tabix/bcftools indexes

See  

https://github.com/samtools/hts-specs/pull/266#discussion_r282908254

and 

https://github.com/macarthur-lab/gnomad_browser/issues/140

Thanks

**Comment by tobiasrausch** [comment]
Hi,

Indeed, since 8 years Delly uses CHROM:POS with INFO:CIPOS for the first breakpoint and INFO/CHR2:INFO/END with INFO:CIEND for the second breakpoint. From an SV point of view that's very consistent because every paired-end/split-read cluster gives rise to 2 breakpoints and INFO:CT clarifies how the two ends are joined and that works for both, intra-chromosomal SVs and inter-chromosomal SVs. I guess, INFO/END and INFO/CIEND don't create issues for intra-chromosomal SVs for tabix/bcftools? I am happy to change the notation for inter-chromosomal SVs but then the specification should clarify how the second chromosome is reported (INFO/CHR2 ?), the second breakpoint position (INFO/POS2 ?) and the confidence interval (INFO/CIPOS2 ?). As previously discussed in this [issue](https://github.com/dellytools/delly/issues/59) I am very reluctant to break-up inter-chromosomal translocations into multiple records because this notion of having one VCF record for each paired-end/split-read cluster is easy to work with and prevents duplicating genotypes across hundreds of samples.

Best, Tobias

**Comment by tobiasrausch** [comment]
Hi,

Delly uses now INFO/CHR2 and INFO/POS2 for inter-chromosomal translocations to be VCF-compliant. For all intra-chromosomal SVs POS and INFO/END is used.

Thanks, Tobias

**Comment by fgvieira** [comment]
Hi Tobias,

are these fixed supposed to be already on version 0.8.2? I used it on a sample and got this SV:
```
2       33141291        BND00002717     T       [1:19212036[T   .       LowQual IMPRECISE;SVTYPE=BND;SVMETHOD=EMBL.DELLYv0.8.2;CHR2=1;END=19212036;PE=2;MAPQ=33;CT=5to5;CIPOS=-460,460;CIEND=-460,460
```

but `tabix` complained that the coordinates were wrong:
```
[E::hts_idx_push] Invalid record on sequence #2: end 19212036 < begin 33141291
```

I raised the [issue](https://github.com/samtools/bcftools/issues/1153) on `bcftools` that said it was a `Delly` bug.

**Comment by tobiasrausch** [comment]
No, I just added it to the GitHub code a couple days ago. I will release a new version soon but there are some other open issues that need to be fixed first. Thanks.
